include/identity_key.h

```diff
@@ -26,13 +26,22 @@ extern "C" {
 #define IDENTITY_KEY_SIZE_BYTES                 (32)
 
 /** @brief Error value when MKEK is missing from the KMU */
-#define ERR_IDENTITY_KEY_MKEK_MISSING           (0x15501)
+#define IDENTITY_KEY_ERR_MKEK_MISSING           (0x15501)
```

Comment:
Document these ones that have been changed too (sorry).

Reply:
Done